igor-web/src/main/groovy/com/netflix/spinnaker/igor/artifacts/ArtifactExtractor.java

```diff
@@ -91,7 +91,7 @@ private JinjaTemplate getTemplateFromProperty(GenericBuild build) {
     return jinjaTemplateService.getTemplate(messageFormat, templateType);
   }
 
-  private boolean parseCustomFormat(Object customFormat) {
+  protected boolean parseCustomFormat(Object customFormat) {
```

Comment:
If this is only for testing, please add `@VisibleForTesting`...except I don't see anything inheriting from ArtifactExtractor so I'm not sure this is necessary.

Reply:
Maybe package-private?

Reply:
Done